exporter/local/fs.go

```diff
@@ -180,7 +180,7 @@ func CreateFS(ctx context.Context, sessionID string, k string, ref cache.Immutab
 			return nil, nil, err
 		}
 		stmtFS := staticfs.NewFS()
-		split := opt.UsePlatformSplit(isMap)
+		split := isMap && !opt.UsePlatformSplit(isMap)
```

Comment:
Could be better to use a different name in here, eg. `addPlatformToName`, as this is slightly different from the platforms being split or not.